tst/CTA.WebForms2Blazor.Tests/Helpers/RuntimeInjectableTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using CTA.WebForms2Blazor.Helpers;
+using NUnit.Framework;
+using Microsoft.CodeAnalysis;
+
+namespace CTA.WebForms2Blazor.Tests.Helpers
+{
+    [TestFixture]
+    public class RuntimeInjectableTests
+    {
+        private const string TestInjectableNamePascalCase = "TestName";
```

Comment:
Is it necessary to test the camel case and the pascal case? These are usually syntax preferences for development rather than actual functional tests no? We can keep them there I was just wondering if there was any other thoughts behind these tests.

Reply:
The reason I chose to include these tests is mainly to make sure that when we reference some form of these runtime injectables we use the correct names. Also following these naming conventions likely benefits the customer